homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -141,6 +145,11 @@ async def async_create_miio_device_and_coordinator(
         device = AirPurifier(host, token)
     elif model.startswith("zhimi.airfresh."):
         device = AirFresh(host, token)
+    # Pedestal fans
+    elif model == MODEL_FAN_P5:
+        device = FanP5(host, token)
+    elif model in MODELS_FAN_MIIO:
+        device = Fan(host, token, model=model)
```

Comment:
`Fan` also supports (I think at least) also the models shown here https://github.com/rytilahti/python-miio/blob/master/miio/fan.py#L12 (p5 being sole miot implementation).

Reply:
`zhimi.fan.sa1`, `zhimi.fan.v2` and `zhimi.fan.v3` added.